birdhouse/config/jupyterhub/jupyterhub_config.py.template

```diff
@@ -187,4 +187,5 @@ blocked_users = {'authtest', '${CATALOG_USERNAME}', 'anonymous'}
 c.Authenticator.blacklist = blocked_users  # v0.9+
 c.Authenticator.blocked_users = blocked_users  # v1.2+
 
+${JUPYTERHUB_CONFIG_EXTENDED}    # noqa
```

Comment:
Why not re-use the existing `JUPYTERHUB_CONFIG_OVERRIDE`?  

If you want to keep both, please leave 2 lines between the 2 variables so that when both are actually in use, they do not look mixed up.

Reply:
I tried doing that, but given that some instances already use `export JUPYTERHUB_CONFIG_OVERRIDE=...` in their `env.local`, which is sourced when runninig `pavics-compose.sh`, any overrides applied along the way get ignored. 

Because of how `JUPYTERHUB_CONFIG_EXTENDED` gets defined using extra newlines, they are already separated by multiple extra lines. I could add more though, or even add comments in the template to separate them explicitly.